### PR TITLE
Update pinnings

### PR DESCRIPTION
**Pull request by xmnlab**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xmnlab** [comment]
with pymapd 0.22, it should also fix https://github.com/omnisci/pymapd/issues/292

File: recipe/meta.yaml [hunk 1]
```diff
@@ -31,12 +31,12 @@ requirements:
     - impyla >=0.15.0
     - lz4
     - multipledispatch >=0.6
-    - numpy >=1.11
-    - pandas >=0.25
+    - numpy >=1.15
+    - pandas >=0.25.3
     - psycopg2
-    - pyarrow >=0.13
+    - pyarrow >=0.15
```
Comment (datapythonista) [inline recipe/meta.yaml:37]: We've got pyarrow 0.13 in the CI builds. Should we update the CI builds, or allow lower versions here?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -31,12 +31,12 @@ requirements:
...
     - pydata-google-auth
-    - pymapd >=0.12.0  # [not py38]
+    - pymapd >=0.22.0
```
Comment (datapythonista) [inline recipe/meta.yaml:39]: Same, 0.21 in the CI (it's commented atm, so may be not important)

File: recipe/meta.yaml [hunk 3]
```diff
@@ -49,7 +49,7 @@ requirements:
     - shapely
     - setuptools
     - sqlalchemy >=1.1
-    - thrift >=0.9.3
+    - thrift >=0.11
```
Comment (datapythonista) [inline recipe/meta.yaml:52]: 0.9.3 in the CI.

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
lgtm. lmk when ready.

**Comment by xmnlab** [comment]
Ok .. just need to rerender and change the build number .. I will let you know when it is ready.

**Review by datapythonista — COMMENTED** [review]
So, if I'm understanding it correctly, before these changes, the time to solve the environment in the build is close to 45 minutes:
```
2020-05-07T12:50:04.7272755Z Collecting package metadata (repodata.json): ...working... done
2020-05-07T13:33:19.5025965Z Solving environment: ...working... done
```
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=157972&view=logs&j=4f922444-fdfe-5dcf-b824-02f86439ef14&t=937c195f-508d-5135-dc9f-d4b5730df0f7&l=387

And with these changes it takes 1 minute:
```
2020-05-07T18:04:36.1978545Z Collecting package metadata (repodata.json): ...working... done
2020-05-07T18:05:35.1478598Z Solving environment: ...working... done
```
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=158182&view=logs&j=4f922444-fdfe-5dcf-b824-02f86439ef14&t=937c195f-508d-5135-dc9f-d4b5730df0f7&l=387

I guess it makes sense to keep consistency with the CI (and I guess we need to update `setup.py` to be consistent with pip). But seems like one of those is the one that makes `pymapd` kill the solver.

CC: @jreback

**Comment by xmnlab** [comment]
@conda-forge-admin, please rerender

**Comment by xmnlab** [comment]
@jreback it is ready! thanks.

**Comment by jreback** [comment]
thanks @xmnlab can you open an issue on the main repo to see if we want to sync these dependencies with the ones in the main repo (e.g. @datapythonista comments)

**Comment by xmnlab** [comment]
sure I will do it. thanks.

**Pull request merged**